jest.config.js

```diff
@@ -25,7 +25,9 @@ module.exports = {
       "<rootDir>/.jest/file-mock.ts",
   },
   setupFilesAfterEnv: ["<rootDir>/.jest/jest-setup.ts"],
-  transformIgnorePatterns: ["node_modules/"],
+  transformIgnorePatterns: [
```

Comment:
Could you test if you still need this with uppy on the dev dependencies, please?